packages/app-accounts/src/modals/Create.tsx

```diff
@@ -323,4 +337,11 @@ export default styled(Create)`
   .accounts--Creator-advanced {
     margin-top: 1rem;
   }
+  .bullet-points {
+    margin-left: 0.3rem;
+    color: rgba(78,78,78,.66);
```

Comment:
can put colors in the styled-themeing file as constants and re-use them here

Reply:
done